pyrenew/convolve.py

```diff
@@ -46,8 +50,8 @@ def new_convolve_scanner(
     -------
     Callable
         A scanner function that can be used with
-        :func:`jax.lax.scan` or
-        :func:`numpyro.contrib.control_flow.scan`
+        [jax.lax.scan][] or
```

Comment:
Prefer these in `monospace`, e.g.
```suggestion
        [`jax.lax.scan`][] or
```

Reply:
This breaks the links.